pkg/kritis/metadata/containeranalysis/containeranalysis.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package containeranalysis
+
+import (
+	"fmt"
+
+	gen "cloud.google.com/go/devtools/containeranalysis/apiv1alpha1"
+	"github.com/grafeas/kritis/pkg/kritis/metadata"
+	"golang.org/x/net/context"
+	"google.golang.org/api/iterator"
+	containeranalysispb "google.golang.org/genproto/googleapis/devtools/containeranalysis/v1alpha1"
+)
+
+const (
+	PkgVulnerability = "PACKAGE_VULNERABILITY"
+	PageSize         = int32(100)
+)
+
+// The ContainerAnalysis struct implements MetadataFetcher Interface.
+type ContainerAnalysis struct {
+	client *gen.Client
+	ctx    context.Context
+}
+
+func NewContainerAnalysisClient() (*ContainerAnalysis, error) {
+	ctx := context.Background()
+	client, err := gen.NewClient(ctx)
+	if err != nil {
+		return nil, err
+	}
+	return &ContainerAnalysis{
+		client: client,
+		ctx:    ctx,
+	}, nil
+}
+
+// GetVulnerabilites get Occurrences of type PACKAGE_VULNERABILITY
```

Comment:
Probably don't want to include the enum value in this comment, since it changed in v1beta1. Maybe just say "GetVulnerabilites gets vulnerability occurrences."

Reply:
thanks!

Reply:
Re: which package has fix avaialbe, we dont care about the fixed package details. The policy only checks if " packages with fixed are allowed or not in a container image or not"

Reply:
Ah okay.